common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -31,6 +34,10 @@ public MsalOAuth2TokenCache(final Context context,
                                 final IAccountCredentialAdapter accountCredentialAdapter,
                                 final List<IShareSingleSignOnState> sharedSsoCaches) {
         super(context);
+        Logger.getInstance().setLogLevel(Logger.LogLevel.VERBOSE);
```

Comment:
The logger level should be set in the application, not in each class. Same with the logcat and pii setting. @addev-ashish  Please confirm.